### ISSUE TITLE
IOError: [Errno 2] No such file or directory: 'README.md'

### ISSUE THREAD

**Issue by mukeshtiwari** [scikit-multilearn/scikit-multilearn#26]
```javascript
hello:scikitlearn mukeshtiwari$ sudo pip install scikit-multilearn
Password:
The directory '/Users/mukeshtiwari/Library/Caches/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
The directory '/Users/mukeshtiwari/Library/Caches/pip' or its parent directory is not owned by the current user and caching wheels has been disabled. check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
Collecting scikit-multilearn
/Library/Python/2.7/site-packages/pip-7.1.2-py2.7.egg/pip/_vendor/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
  Downloading scikit-multilearn-0.0.1.tar.gz (32.5MB)
    100% |████████████████████████████████| 32.5MB 11kB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/private/tmp/pip-build-BguQD2/scikit-multilearn/setup.py", line 11, in <module>
        long_description=open('README.md').read(),
    IOError: [Errno 2] No such file or directory: 'README.md'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/tmp/pip-build-BguQD2/scikit-multilearn

**Comment by mukeshtiwari** [comment]
```javascript
hello:scikitlearn mukeshtiwari$ uname -a
Darwin hello.local 14.0.0 Darwin Kernel Version 14.0.0: Fri Sep 19 00:26:44 PDT 2014; root:xnu-2782.1.97~2/RELEASE_X86_64 x86_64

**Comment by niedakh** [comment]
this is fixed in head, I'll do a release this week :)

**Comment by tsalo** [comment]
I'm getting this error while trying to install. Was the release pushed to PyPI?

**Comment by niedakh** [comment]
I've postponed the release because I needed to finish one journal paper asap in january, I'm cleaning up the code, and finishing support for sparse matrices in classifiers. I'll merge some bugfixes from other contributors as well and will release this week, I promise!

**Comment by niedakh** [comment]
Fixed in 0.0.3, thank your for your help!

**Comment by akhilgentoo** [comment]
Traceback (most recent call last):
  File "setup.py", line 22, in <module>
    long_description=read('README'),
  File "setup.py", line 9, in read
    return open(os.path.join(os.path.dirname(__file__), fname)).read()
IOError: [Errno 2] No such file or directory: 'README'
